easybuild/easyconfigs/a/ALLPATHS-LG/ALLPATHS-LG-52488-foss-2016a.eb

```diff
@@ -0,0 +1,37 @@
+##
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+#
+# Copyright:: Copyright 2012-2013 The Cyprus Institute
+# Authors::   Andreas Panteli <a.panteli@cyi.ac.cy>, Thekla Loizou <t.loizou@cyi.ac.cy>, 
+#             George Tsouloupas <g.tsouloupas@cyi.ac.cy>
+# License::   MIT/GPL
+#
+##
+# 52488 modified by:
+# Adam Huffman
+# The Francis Crick Institute
+
+easyblock = 'ConfigureMake'
+
+name = 'ALLPATHS-LG'
+version = '52488'
+
+homepage = 'http://www.broadinstitute.org/software/allpaths-lg/blog/'
+description = """ALLPATHS-LG, the new short read genome assembler."""
+
+toolchain = {'name': 'foss', 'version': '2016a'}
+
+# source is moved over time, hence multiple URLs below
+source_urls = [
+    'ftp://ftp.broadinstitute.org/pub/crd/ALLPATHS/Release-LG/latest_source_code',
+    'ftp://ftp.broadinstitute.org/pub/crd/ALLPATHS/Release-LG/latest_source_code/2013-06',
+    'ftp://ftp.broadinstitute.org/pub/crd/ALLPATHS/Release-LG/latest_source_code/2013/2013-06',
```

Comment:
@boegel - yes, I've removed them now, leaving in the note, so they can be added later as required